include/engine/guidance/route_step.hpp

```diff
@@ -60,6 +60,7 @@ inline IntermediateIntersection getInvalidIntersection()
 struct RouteStep
 {
     unsigned name_id;
+    bool is_segregated;
```

Comment:
would prefer this being located close to the maneuver, or even being made part of the maneuver. After all it is referring to the turn, not to the step

Reply:
It make sense if you mean StepManeuver which is a part of RouteStep, right?

Reply:
Yes. I was referring to the StepManeuver